Docs/developer_guide/modules/transforms.md

```diff
@@ -17,6 +17,24 @@
   - Avoid inverse non-linear transforms: make sure that non-linear transforms are only set as FromParent
 - Transforms module in Slicer shows linear transform matrix values "to parent" (modeling convention) in RAS coordinate system. Therefore to retrieve the same values from an ITK transforms as shown in Slicer GUI, one has switch between RAS/LPS and modeling/resampling. See example [here](../script_repository.md#convert-between-itk-and-slicer-linear-transforms).
 
+## Displacement field files
+
+Slicer can store displacement field transforms (or "grid transforms" in VTK naming convention) in 3D NRRD or NIFTI format; or sequence of transforms in NRRD format. Image file header fields are used to distinguish the displacement field transforms form regular images and specify how to correctly interpret these images as transforms.
+
+Requirements for NRRD files to be correctly recognized and loaded as displacement field transform:
+- `kinds` field is set to:
+  - `vector domain domain domain list` for single 3D displacement field transform
+  - `vector domain domain domain list` for displacement field sequence
+- `measurement frame` is set and it defines the coordinate system the displacement vectors are defined in.
+- custom `intent_code` field is set to `1006`. This convention originates from the NIFTI file format and it makes it easy to detect images as displacement fields when using ITK file reader.
+- custom `displacement field type` is set to:
+  - `resampling` for ITK convention, useful for transforming images, in the Slicer transform tree specifies the transform "from parent"
+  - `modeling` (useful for transforming points, markups, models; in the Slicer transform tree specifies the transform "to parent")
+
+Requirements for NIFTI files to be correctly recognized and loaded as displacement field transform:
+- `intent_code` field is set to 1006 (`NIFTI_INTENT_DISPVECT`)
+- displacement vector values are stored in RAS coordinate system
+
```

Comment:
It would be nice to include a specific example of each type of nrrd header just to be concrete.